lib/kramdown/parser/kramdown/codeblock.rb

```diff
@@ -32,7 +32,7 @@ def parse_codeblock
 
 
       FENCED_CODEBLOCK_START = /^~{3,}/
-      FENCED_CODEBLOCK_MATCH = /^((~){3,})\s*?(\w+)?\s*?\n(.*?)^\1\2*\s*?\n/m
+      FENCED_CODEBLOCK_MATCH = /^((~){3,})\s*?(\w+-)?\s*?\n(.*?)^\1\2*\s*?\n/m
```

Comment:
Sure. Must have been blind. Fixed that, but forgot to push. :smile: But there's some more code which needs to be changed. Somewhere "you" may split the html-class-name. Only parts of the same go to `class="lang"`